### ISSUE TITLE
Track the number of commits dropped per engine roll 

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#63115]
quote from @yjbanov 

> The reason this number is important is because when we drop a lot of commits, there's a much higher chance that it will have to be rolled back, and the extra size of the drop makes the rollback process disproportionately more difficult (e.g. it requires a longer bisection). Large number of commits also means higher latency between an engine change and its landing in the framework.

This should be doable when we parse/collect flutter rollers metrics.

**Comment by godofredoc** [comment]
@keyonghan did we ended up adding this dashboard?

**Comment by keyonghan** [comment]
No. This is not implemented ever. Un-assigning myself as I am not working on this.
